dotnet/MomentoExamples/MomentoApplicationPresignedUrlNet45/Program.cs

```diff
@@ -0,0 +1,74 @@
+﻿using System;
```

Comment:
Curious - Is there any difference between this example and the .NET 6.0 one?

Reply:
Only the part to get expiry epoch is different:

```
5c5
< using MomentoSdk;
---
> using MomentoSdkDotnet45;
54c54
<             uint expiryEpochSeconds = (uint) DateTimeOffset.UtcNow.AddMinutes(URL_TTL_MINUTES).ToUnixTimeSeconds();
---
>             uint expiryEpochSeconds = (uint) DateTimeOffset.UtcNow.AddMinutes(URL_TTL_MINUTES).Subtract(new DateTime(1970, 1, 1)).TotalSeconds;
```

The method `ToUnixTimeSeconds` used in .NET 6.0 is not supported on 4.5, and the code used in 4.5 does not work on my mac but works when tested on Windows machine.  I didn't spend more time looked into why or what methods worked on both.